application/src/main/java/eu/europa/ec/fisheries/uvms/activity/message/consumer/bean/ActivityMessageConsumerBean.java

```diff
@@ -103,43 +81,20 @@ public void onMessage(Message message) {
                     saveReport(textMessage);
                     break;
                 case GET_FISHING_TRIPS:
-                    getFishingTrips(textMessage);
-                    break;
                 case GET_NON_UNIQUE_IDS:
-                    getNonUniqueIds(textMessage);
-                    break;
                 case GET_FLUX_FA_QUERY:
```

Comment:
Is there a point in having cases that fall through to default? Can't you just have the default? Because it's the default?